test/testpkg.jl

```diff
@@ -39,4 +39,10 @@ function qux(a::Dict, b::Vector)
     end
 end
 
+function tagscheck(x)
+    for package_tag in [1, 2, 3]
```

Comment:
What's this supposed to be checking?

Reply:
oh yeah I should change that, good catch